### DISCUSSION TITLE
yarn version-git-message does not exist on yarn 2?

### DISCUSSION THREAD

**Discussion by dopecodez** [yarnpkg/berry#2438] [General]
So, i am a collaborator in [np](https://github.com/sindresorhus/np) which is one of the most popular libraries out there for publishing npm packages.

We are using `yarn version --new-version <version>` internally to update the `package.json` for projects. As per a [new feature request](https://github.com/sindresorhus/np/issues/590), the community was trying to implement a feature where a user can supply a custom commit message to this version update commit by `yarn`(or `npm`).

`npm` pretty much only needs us to use the '-m' flag for this, and classic `yarn` had `yarn config set version-git-message "v%s"` to set a custom message for the version commit. However, looking through the docs for the `berry` version of yarn does not seem to have a feature for this. 

Does something like this exist and the documentation was missed somehow? Or is this is a missing feature we should think about implementing?

**Comment by arcanis** [comment]
Our current implementation of `yarn version` doesn't create a git commit at all, so this setting doesn't have a parallel. It's however something we're considering adding back in 3.x in some capacity, so I'll keep that in mind.

**Reply by dopecodez** [reply]
Thanks for the reply @arcanis . If you're not making a commit, missing this setting does make sense. So we should not be expecting yarn 2 to be supporting this at any stage, right?
